collector/kernel/process_handler.h

```diff
@@ -41,40 +37,13 @@
 //#define DEBUG_TGID 1
 #endif // NDEBUG
 
-#define ENUM_NAME ProcPath
-#define ENUM_TYPE std::uint8_t
-#define ENUM_ELEMENTS(X)                                                                                                       \
-  X(stat, 0)                                                                                                                   \
-  X(io, 1)                                                                                                                     \
-  X(status, 2)
-#include <util/enum_operators.inl>
-
 class ProcessHandler {
 public:
-  struct CpuMemIoSettings {
-    // minimum and maximum cpu/mem/io batch size
-    std::size_t min_batch_size;
-    std::size_t max_batch_size;
-    // maximum time budget per cpu/mem/io micro batch
-    StopWatch<>::duration poll_budget;
-    // poll count between cpu/mem/io batch chunks
-    std::size_t batch_poll_count_cooldown;
-  };
-
   ProcessHandler(
-      ::flowmill::ingest::Writer &writer,
-      ::flowmill::kernel_collector::Index &collector_index,
-      ProbeHandler &probe_handler,
-      ebpf::BPFModule &bpf_module,
-      logging::Logger &log,
-      // if null, disables cpu/mem/io collection
-      CpuMemIoSettings const *cpu_mem_io_settings);
+      ::flowmill::ingest::Writer &writer, ::flowmill::kernel_collector::Index &collector_index, logging::Logger &log);
```

Comment:
just verifying since this looks a bit odd -- if you haven't run clang-format, please run it so we can keep all the code in the same standard.. this repo doesn't have auto formatting yet.

Reply:
clang format did that :)